src/core/fileinfo.cpp

```diff
@@ -359,11 +359,11 @@ bool FileInfo::isExecutableType() const {
         if(isNative() && (mode_ & (S_IRUSR|S_IRGRP|S_IROTH))) {
             if(isShortcut() && !target_.empty()) {
                 /* handle shortcuts from desktop to menu entries:
-                   first check for entries in /usr/share/applications and such
+                   first check for entries in ${PREFIX}/share/applications and such
                    which may be considered as a safe desktop entry path
                    then check if that is a shortcut to a native file
                    otherwise it is a link to a file under menu:// */
-                if (!g_str_has_prefix(target_.c_str(), "/usr/share/")) {
+                if (!g_str_has_prefix(target_.c_str(), "${PREFIX}/share/")) {
```

Comment:
What is `${PREFIX}` and how would it be evaluated in compile/run-time?

Reply:
On Linux is /usr
on NetBSD is /usr/pkg
on FreeBSD is /usr/local

Because of these differences hard-coding it causes portability issues. I don't have a Linux machine right now but on NetBSD ${PREFIX} will be evaluated as /usr/pkg, in this case /usr/pkg/share will be the result.
I can't test on Linux, that's why I asked to please review.

Reply:
> ${PREFIX} will be evaluated as /usr/pkg

Who and when will it evaluate?